Gruntfile.js

```diff
@@ -2,139 +2,17 @@
 'use strict';
 
 module.exports = function(grunt) {
-    require('load-grunt-tasks')(grunt);
-
     grunt.initConfig({
-        config: {
```

Comment:
I’m a fan of keeping everything in the root _Gruntfile.js_ if it isn’t super long (<200 lines). But whatevs. Matches our other repos. `¯\_(GRUNTFILE)_/¯`